### DISCUSSION TITLE
Option to choose payment method during purchase

### DISCUSSION THREAD

**Discussion by ssiwm** [gameserverapp/Platform#459] [Suggestions]
We currently need to create "Supporter Tiers" clones so that players can choose which method they want to pay.  My suggestion is that they can choose it at the end of the order

**Comment by mvaessen** [comment]
Inviting people to share their opinion on the proposed solution. 
See https://discord.com/channels/315049169071112193/997450569688035358/1296462576116305982

**Comment by LegitGammin** [comment]
Is it possible to add cashapp payment method also?

**Reply by mvaessen** [reply]
for that you need to add another suggestion, if it isn't already on there.
This suggestion is specifically about selecting payment methods during the ordering process, vs the current situation where you need to create a new pack for each provider.

**Comment by mvaessen** [comment]
Suggestion from Birdman: add option to set preferred / default provider. This would auto-select the preferred option

**Comment by mvaessen** [comment]
This is now available
